engine/src/main/java/org/entando/entando/web/health/HealthController.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2015-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.web.health;
+
+import org.entando.entando.aps.system.services.health.IHealthService;
+import org.entando.entando.web.health.model.HealthResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/health")
+public class HealthController {
+
+    private final String STATUS_UP = "UP";
+    private final String STATUS_DOWN = "DOWN";
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    private IHealthService healthService;
+
+    @Autowired
+    public HealthController(IHealthService healthService) {
+        this.healthService = healthService;
+    }
+
+    @GetMapping
+    public ResponseEntity<HealthResponse> isHealthy() {
+
+        logger.debug("health check");
+
+        boolean healthy = this.healthService.isHealthy();
+
+        return new ResponseEntity<>(new HealthResponse(healthy ? STATUS_UP : STATUS_DOWN), HttpStatus.OK);
```

Comment:
`STATUS_UP` and `STATUS_DOWN` are never used, since when status is unhealthy an exception is thrown.
Either remove this check and always return `STATUS_UP`, or remove Exception thrown, send false when is not healthy and handle true/false value in here.

Everything else looks good to me!

Reply:
I prefer the second option, since it is cleaner.
When the status is unhealthy you're handling it as an Error response, but actually nothing wrong has happened. Returning false is a normal response in the flow and responsibility of this endpoint.

Reply:
Ok it makes sense. @Kerruba if you agree too I proceed to apply the requested changes

Reply:
@firegloves I agree with @ffleandro